package.json

```diff
@@ -58,6 +58,7 @@
   "devDependencies": {
     "babel-eslint": "^6.0.2",
     "babel-loader": "^6.2.4",
+    "babel-plugin-transform-react-remove-prop-types": "0.2.5",
```

Comment:
Can you move it to "dependencies" ? Currently project builds automatically on user side after installing, and it will break this setup